src/data/mutable_data.rs

```diff
@@ -39,7 +39,7 @@ pub struct MutableData {
     version: u64,
     /// Contains a set of owners which are allowed to mutate permissions.
     /// Currently limited to one owner to disallow multisig.
-    owners: BTreeSet<PublicKey>,
+    owners: BTreeSet<PublicKeys>,
```

Comment:
Not correct at all - this is what was explicitly noted in the Friday Gossip on safe-crypto that we don't want encryption keys but *_only _* the signing keys and safe_crypto has that facility to hand over only the signing keys